pkg/webhook/default_server/sidecarset/mutating/sidecarset_create_update_handler.go

```diff
@@ -160,12 +160,12 @@ func (h *SidecarSetCreateHandler) Handle(ctx context.Context, req types.Request)
 
 	switch req.AdmissionRequest.Operation {
 	case v1beta1.Create, v1beta1.Update:
-		setDefaultSidecarSet(copy)
+		SetDefaultSidecarSet(copy)
 		if err := setHashSidecarSet(copy); err != nil {
 			return admission.ErrorResponse(http.StatusInternalServerError, err)
 		}
 	}
-
+	klog.V(3).Infof("sidecarset after mutating: %v", util.DumpJSON(copy))
```

Comment:
should this log be removed or increase its log level

Reply:
I am OK with increasing the lvl.